.github/workflows/client-exports.yaml

```diff
@@ -91,17 +94,18 @@ jobs:
           godot --headless --verbose --export-release "Web" ../build/web/index.html
       - name: Upload Artifact
         uses: actions/upload-artifact@v1
+        if: github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
I hope this works, was the response of chat-gpt, I didn't had any sample in my hand, if doesn't work, I will modify and fix it